components/Tabs/Ficha.js

```diff
@@ -14,59 +23,82 @@ export default function Ficha() {
           <div className="flex mt-4">
             <span className="flex text-neutral-800 text-sm font-semibold flex-1">
               Primaria
-              <Completed className="ml-3" />
+              {candidate.educacion.primaria ? (
+                <Completed className="ml-3" />
+              ) : (
+                <Incompleted className="ml-3" />
+              )}
             </span>
             <span className="flex text-neutral-800 text-sm font-semibold flex-1">
               Secundaria
-              <Incompleted className="ml-3" />
+              {candidate.educacion.secundaria ? (
+                <Completed className="ml-3" />
+              ) : (
+                <Incompleted className="ml-3" />
+              )}
             </span>
           </div>
         </div>
         <div className="px-6 pt-4 pb-6">
           <h4 className="text-xs text-neutral-400 font-semibold">
             EDUCACIÓN UNIVERSITARIA
```

Comment:
I wonder what will be shown if the candidate doesn't have a college education 🤔 cc @ederrengifo

Reply:
Also I just noticed that we have postgrado data that we are not showing 😅

Reply:
I suppose that postgraduate degrees should be shown in university education as well ☝️